channeldb/channel.go

```diff
@@ -188,6 +195,10 @@ var (
 	// in the state CommitBroadcasted.
 	ErrNoCloseTx = fmt.Errorf("no closing tx found")
 
+	// ErrNoShutdownInfo is returned when no shutdown info has been
+	// persisted for a channel.
+	ErrNoShutdownInfo = fmt.Errorf("no shutdown info")
```

Comment:
nit: why not `errors.New`?

Reply:
honestly was just to match the rest here. But happy to change it :)